config/initializers/prawn.rb

```diff
@@ -0,0 +1,13 @@
+# Force the use of fallback fonts in Prawn, either by providing the argument `fallback_fonts`
+# or by using the Export::Pdf::Document class instead of Prawn::Document directly.
+Prawn::Document.prepend(Module.new do
+  def initialize(options = {}, &_block)
```

Comment:
Einfach für mich zum Verständnis: Wenn wir uns schon hier in Prawn hineinhacken, warum setzen wir dann nicht einfach Noto als Fallback Font? Und in Wagons die andere Fonts verwenden kann man eine spezifischere fallback font setzen?

Reply:
Wir erzwingen damit vor allem, dass unsere eigene `Export::Pdf`-Klasse verwendet wird, damit wir allgemein mehr Konsistenz haben. Der Fallback-Font ist aus meiner Sicht nur der Trigger für die Anpassung und ein praktischer Marker, um das herauszufinden.